lua/gk.lua

```diff
@@ -15,8 +15,50 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local fib_entries = {
+		{
+			["ip_addr"] = "187.73.32.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["ip_addr"] = "187.73.33.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_DROP,
+		},
+	}
+
+	local grantors = {
+		{
+			["gw_id"] = -1,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["gw_id"] = 0,
+			["grantor"] = "3ffe:2501:200:3::1",
+		},
+	}
+
+	local gateways = {
```

Comment:
I have said it before, and I repeat it here: we should not require parameters in the configuration file that can be computed automatically because this only adds cost to maintain Gatekeeper. We should only require admins to enter the FIB entries needed, our code should find all other information needed from that input.